src/components/Toolbelt/Toolbelt.test.js

```diff
@@ -49,5 +56,14 @@ describe('<ToolBelt />', () => {
         true
       )
     })
+
+    test('marks the shovel selected for field mode MINE', async () => {
```

Comment:
This function is `async`, but it has no `await`s. Should `async` be removed?

Reply:
yup! also came from the testing adventure. user input with testing lib requires async/await behavior, but i should have removed this with the removal of the click action in the test